code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -222,6 +226,9 @@
 		s_store = null
 		if(!QDELETED(src))
 			update_inv_s_store()
+	. = ..(I, force, newloc, no_move, invdrop, was_thrown) //See mob.dm for an explanation on this and some rage about people copypasting instead of calling ..() like they should.
```

Comment:
```suggestion
	. = ..(args) //See mob.dm for an explanation on this and some rage about people copypasting instead of calling ..() like they should.
```
But what is stopping us from just doing this :)

Reply:
the fact that it now just deletes the items instead of putting it in our hands